src/Moryx.Drivers.Mqtt/MqttTopic.cs

```diff
@@ -119,8 +141,21 @@ public string Identifier
         [Display(Name = nameof(Strings.MqttTopic_ResponseTopic), Description = nameof(Strings.MqttTopic_ResponseTopic_Description), ResourceType = typeof(Strings))]
         public string ResponseTopic { get; set; }
 
+        /// <summary>
+        /// Marks if messages send by this topic should be marked as retained by default.
+        /// Messages can implment IRetainAware to overwrite the default.
+        /// </summary>
+        [EntrySerialize, DataMember, DefaultValue(false)]
```

Comment:
Please do not use default for bool. True cannot work and false is already the default.